Tests/ViewInspectorTests/SwiftUI/TreeViewTests.swift

```diff
@@ -42,3 +52,115 @@ final class GlobalModifiersForTreeView: XCTestCase {
         XCTAssertNoThrow(try sut.inspect().emptyView())
     }
 }
+
+// MARK: - VariadicViewScreen
+
+@available(iOS 13.0, macOS 10.15, *)
+private struct VariadicViewScreen: View {
+    @State var opacity: Double = 1.0
+
+    var body: some View {
+        _VariadicView.Tree(VariadicViewScreenOpacityRoot(opacity: opacity),
```

Comment:
I'm curious if you're using `_VariadicView` in the production code? It seems to be the type resolved after [inlining stack views](https://movingparts.io/variadic-views-in-swiftui). I'm ok with this test, though, for the sake of clarity and conciseness, and if it's possible, could we refactor this piece to use a stack view that unfolds into `_VariadicView`?

Reply:
Yes. We do have a custom View that calls the `_VariadicView.Tree` directly from its `body` property implementation.

The goal I had in mind for this test is exercising the direct call. The Layout test covers the case of a `View` that unfolds into a `_VariadicView.Tree`.

I'm also okay removing this test if you prefer.

Reply:
One thing I'm having a hard time understanding is that modifying the test to the following code, I get a `VStack` with an underlying `_VariadicView.Tree`, but the code  in TreeView.swift is not called. And the button is found.

```swift
    func testVariadicViewTree() throws {
        let button = try VStack {
            Text("Click the button to change the opacity.")
            Button(action: {
                print("test")
            }, label: {
                Text("Change opacity")
            })
        }.inspect().find(button: "Change opacity")
        XCTAssertNotNil(button)
    }
```

```
▿ Content
  ▿ view : VStack<TupleView<(Text, Button<Text>)>>
    ▿ _tree : Tree<_VStackLayout, TupleView<(Text, Button<Text>)>>
      ▿ root : _VStackLayout
        ▿ alignment : HorizontalAlignment
          ▿ key : AlignmentKey
            - bits : 5381443304
        - spacing : nil
      ▿ content : TupleView<(Text, Button<Text>)>
        ▿ value : 2 elements
          ▿ .0 : Text
            ▿ storage : Storage
              ▿ anyTextStorage : <LocalizedTextStorage: 0x7ff7e20c2320>
            - modifiers : 0 elements
          ▿ .1 : Button<Text>
            ▿ _label : Text
              ▿ storage : Storage
                ▿ anyTextStorage : <LocalizedTextStorage: 0x7ff7e20c4a50>
              - modifiers : 0 elements
            - action : (Function)
  ▿ medium : Medium
    - viewModifiers : 0 elements
    - transitiveViewModifiers : 0 elements
    - environmentModifiers : 0 elements
    - environmentObjects : 0 elements
```

Reply:
FYI: I have removed the test that explicitly uses `_VariadicView.Tree` so it does not cause confusion on what the code is supposed to cover.